doc/model_persistence.rst

```diff
@@ -123,3 +123,33 @@ not help in production when performance is critical.
 To convert scikit-learn model to PMML you can use for example `sklearn2pmml
 <https://github.com/jpmml/sklearn2pmml>`_ distributed under the Affero GPLv3
 license.
+
+A more secure format: `skops`
```

Comment:
I am wondering if it should be a section 9.1.2 just after the section regarding the security and maintainability limitation.

Reply:
I see what you mean. I moved it to be the 9.1.2 section, I agree it's a more natural flow reading from the top.